DDCore/src/segmentations/SquareGrid.cpp

```diff
@@ -0,0 +1,105 @@
+/*
```

Comment:
License notice missing

Reply:
Added in commit [9b7833c](https://github.com/sebouh137/DD4hep/commit/9b7833c04431f588d29e13489c6ae65902e82534)